Entity/Subscription.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+/**
+ * Auto generated by MySQL Workbench Schema Exporter.
+ * Version 3.0.2 (doctrine2-annotation) on 2016-04-29 10:04:43.
+ * Goto https://github.com/johmue/mysql-workbench-schema-exporter for more
+ * information.
+ */
+
+namespace Integrated\Bundle\SubscriptionBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Integrated\Bundle\SubscriptionBundle\Entity\Subscription
+ *
+ * @ORM\Entity()
+ * @ORM\Table(name="subscription", indexes={@ORM\Index(name="fk_Subscription_SubscriptionType1_idx", columns={"`type`"})})
+ */
+class Subscription
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="UUID")
+     * @ORM\Column(type="string", length=36)
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(name="`type`", type="string", length=36)
+     */
+    protected $type;
+
+    /**
+     * @ORM\Column(type="string", length=45, nullable=true)
+     */
+    protected $relation;
+
+    /**
+     * @ORM\Column(type="string", length=45, nullable=true)
+     */
+    protected $contact;
+
+    /**
+     * @ORM\Column(type="decimal", precision=6, scale=2, nullable=true)
+     */
+    protected $customprice;
+
+    /**
+     * @ORM\Column(type="decimal", precision=6, scale=2, nullable=true)
+     */
+    protected $discountprice;
+
+    /**
+     * @ORM\Column(type="decimal", precision=5, scale=3, nullable=true)
+     */
+    protected $discountpercentage;
+
+    /**
+     * @ORM\Column(name="`status`", type="string", length=255, nullable=true)
+     */
+    protected $status;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="SubscriptionType", inversedBy="subscriptions")
+     * @ORM\JoinColumn(name="`type`", referencedColumnName="id", nullable=false)
+     */
+    protected $subscriptionType;
+
+    public function __construct()
+    {
+    }
+
+    /**
+     * Set the value of id.
```

Comment:
unnecessary addition, you only need to add a description if there is some logic that needs explaination. In this case it is clear this function sets the value of id. This counts for all the getters and setters without any custom logic